scripts/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+    "extends": "../tsconfig.json",
+    "compilerOptions": {
+        "noEmit": true,
+        "lib": ["dom", "es2016"]
```

Comment:
Nice catch. 

Yah, I think that is a bit overkill. The `scripts/tsconfig.json` is kind of already using the `./tsconfig.json` as a base config.